addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -30,7 +30,7 @@
                     </t>
 
                     <t t-name="JournalTop">
-                        <div class="o_kanban_card_header">
+                        <div t-attf-class="o_kanban_card_header #{kanban_color(1)}">
```

Comment:
Why kanban_color(0) became kanban_color(1) ? Actually where does it come from ? Does not seem to be a field. Is it ?